sway-lib-std/src/math.sw

```diff
@@ -51,6 +51,36 @@ pub trait Power {
     fn pow(self, exponent: u32) -> Self;
 }
 
+
+impl Power for u256 {
+    /// Raises self to the power of `exp`, using exponentiation by squaring.
+	///
+	/// # Panics
+	///
+	/// Panics if the result overflows the type.
+    fn pow(self, exp: u32) -> Self {
```

Comment:
Here as well, `exponent` for the sake of consistency with other `Power` impls.

Reply:
Done.